recipes/legit/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set packagename = 'legit' %}
+{% set version = "0.2.0" %}
+{% set md5 = "5df3cd402b0de8f0251a87904b5eb88c" %}
+
+package:
+  name: {{ packagename }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/kennethreitz/legit
+  git_tag: v{{ version }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - legit = legit.cli:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - clint >=0.4.1
+    - gitpython >=0.3.4
+    - six >=1.9.0
+
+test:
+  # Some package might need a `test/commands` key to check CLI.
+  imports:
+    - {{ packagename }}
+
+  commands:
+    - cd {{ environ.SRC_DIR }} && legit help
```

Comment:
Oh, I see it now.  environ.SRC_DIR is totally not guaranteed to exist at test time.  You should instead clone a simple repo as part of your test.  You may want to turn to creating run_test.bat/run_test.sh, or run_test.py if you know how to clone/create a repo from python.  Feel free to use legit itself if that helps.